src/NavBrand.js

```diff
@@ -0,0 +1,36 @@
+import React, { cloneElement } from 'react';
+import BootstrapMixin from './BootstrapMixin';
+import classNames from 'classnames';
+
+const NavBrand = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    navbar: React.PropTypes.bool,
+    navbrand: React.PropTypes.bool
```

Comment:
For consistency with how we do dropdowns, this should be something like e.g. `bsRole: 'brand'`. cc @jquense 

https://github.com/react-bootstrap/react-bootstrap/blob/09fe7a54cbb1461f1988ec321ef8227830afc698/src/DropdownToggle.js#L40